packages/@uppy/status-bar/src/Components.tsx

```diff
@@ -48,15 +64,26 @@ function UploadBtn (props) {
   )
 }
 
-function RetryBtn (props) {
+interface RetryBtnProps<M extends Meta, B extends Body> {
+  i18n: Uppy<M, B>['i18n']
```

Comment:
We have an exported `I18n` type in `Translator.ts`.

Reply:
Good to know, especially for interfaces that do not need generics. Not sure I want to bother modifying all of those, hopefully it should be perfectly equivalent, no?

Reply:
Should be a single global search/replace narrowed down to your package?

Reply:
It was more work than that, because I had to remove the unused generics. Anyway that's done